### ISSUE TITLE
[TPU] gcloud error when `sky down` a stopped TPU node

### ISSUE THREAD

**Issue by Michaelvll** [skypilot-org/skypilot#1159]
I encountered the following issue by using the following commands:
```
sky launch -c test-tpu --gpus=tpu-v3-8 '' -y
sky stop test-tpu
sky down test-tpu -y
```
```
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] Failed to terminate test-tpu. If you want to ignore this error and remove the cluster from the status table, use `sky down --purge`.
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] **** STDOUT ****
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493]
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] **** STDERR ****
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] ERROR: (gcloud.compute.instances.delete) argument INSTANCE_NAMES [INSTANCE_NAMES ...]: Must be specified.
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] Usage: gcloud compute instances delete INSTANCE_NAMES [INSTANCE_NAMES ...] [optional flags]
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493]   optional flags may be  --delete-disks | --help | --keep-disks | --zone
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493]
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493] For detailed information on this command and its flags, run:
E 09-07 23:33:31 cloud_vm_ray_backend.py:2493]   gcloud compute instances delete --help
```

**Comment by Michaelvll** [comment]
This error seems to be similar as #1977

**Comment by Michaelvll** [comment]
This should be fixed with #2199.
Tested with the commands in description.
